.evergreen/setup-remote.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -eu
+
+source secrets-export.sh
+
+if [ -z "${DIR:-}" ]; then
+    echo "Must give a target dir!"
+    exit 1
+fi
+
+# Get the correct remote URI.
+case $DIR in
+    llama-index-python-kvstore)
+        MONGODB_URI=$LLAMA_INDEX_MONGODB_URI
+    ;;
+    semantic-kernel-python)
+        MONGODB_URI=$SEMANTIC_KERNEL_MONGODB_URI
+    ;;
+    semantic-kernel-csharp)
+        MONGODB_URI=$SEMANTIC_KERNEL_MONGODB_URI
```

Comment:
Out of scope for this PR:
Could it be that sharing the URI could be leading to some failures in index definitions?

Reply:
I don't think so, given they're using different DBs.